src/main/java/com/andersen/hw/controller/TicketController.java

```diff
@@ -0,0 +1,52 @@
+package com.andersen.hw.controller;
+
+import com.andersen.hw.dto.TicketDtoOut;
+import com.andersen.hw.model.Ticket;
+import com.andersen.hw.service.TicketService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RequestMapping("/tickets")
+@RestController
+@RequiredArgsConstructor
+public class TicketController {
+    private final TicketService ticketService;
+
+    @PostMapping
+    void createTicket(@RequestBody Ticket ticket) {
```

Comment:
all methods in controller have to be with `public`. if you need to make them with other type of access modifier - you need to have a reason for that

Reply:
done